Anytype/Sources/PresentationLayer/ObjectTypeSearch/ObjectTypeSearchInteractor.swift

```diff
@@ -0,0 +1,33 @@
+import Services
+import AnytypeCore
+
+
+final class ObjectTypeSearchInteractor {
```

Comment:
Really needed interactor?

Reply:
Yep, it will be more robust in the near future